doc/classes/AnimatedSprite2D.xml

```diff
@@ -53,7 +54,9 @@
 			The texture's drawing offset.
 		</member>
 		<member name="playing" type="bool" setter="set_playing" getter="is_playing" default="false">
-			If [code]true[/code], the [member animation] is currently playing. Setting this property to [code]false[/code] is the equivalent of calling [method stop].
+			If [code]true[/code], the [member animation] is currently playing. Setting this property to [code]false[/code] pauses the current animation.
+			[b]Note:[/b] Changing this property preserves the current frame's elapsed time and the [code]backwards[/code] parameter (see [method play]).
+			[b]Note:[/b] After a non-looped animation finishes, the property still remains [code]true[/code].
```

Comment:
```suggestion
			[b]Note:[/b] After a non-looping animation finishes, the property still remains [code]true[/code].
```

Docs aside, this behavior is pretty confusing.

Reply:
There may be a reason, but that still means that the property doesn't actually return what we claim it returns, if a finished animation would still report as playing 🙃 Well, no matter if this isn't a part of this PR anyway.